README.md

```diff
@@ -183,7 +183,7 @@ CASSANDRA_EXTENDED_TOKEN_AWARE_POLICY | Enable an extended token aware policy fo
 CASSANDRA_LATENCY_AWARE_POLICY | Enable a latency awareness policy, which collects the latencies of the queries for each Cassandra node and maintains a per-node latency score (an average). | `false`
 CASSANDRA_CUSTOM_RETRY_POLICY | Enable the use of a customized retry policy for your Pega Platform deployment. After enabling this policy in your deployment configuration, Cassandra queries that timeout will be retried. The number of retries may be configured using the dynamic system setting (DSS): dnode/cassandra_custom_retry_policy/retryCount. If not configured, queries will be retried once. | `false`
 CASSANDRA_SPECULATIVE_EXECUTION_POLICY | Enable the speculative execution policy for retrieving data from your Cassandra service. When enabled, the Pega Platform will send a query to multiple nodes in your Cassandra service and process the first query response. This provides lower perceived latencies at the client, but puts greater load on your Cassandra service. | `false`
-CASSANDRA_JMX_METRICS_ENABLED | Enable reporting of DDS SDK metrics to a Java Management Extension (JMX) format for use by your organization to monitor your Cassandra service. Setting this property `false` will disable metrics being exposed via the JMX interface; disabling also limits the metrics being collected using the DDS landing page. | `true`
+CASSANDRA_JMX_METRICS_ENABLED | Enable reporting of DDS SDK metrics to a Java Management Extension (JMX) format for use by your organization to monitor your Cassandra service. Setting this property `false` disables metrics being exposed via the JMX interface; disabling also limits the metrics being collected using the DDS landing page. | `true`
```

Comment:
please review this...don't use via...it's latin, not English ;-)>>
CASSANDRA_JMX_METRICS_ENABLED | Enable reporting of DDS SDK metrics to a Java Management Extension (JMX) format for use by your organization to monitor your Cassandra service. Setting this property `false` disables metrics being exposed through the JMX interface; disabling also limits the metrics being collected using the DDS landing page. | `true`